src/cpp/include/openvino/genai/tokenizer.hpp

```diff
@@ -84,5 +85,28 @@ class OPENVINO_GENAI_EXPORTS Tokenizer {
     std::shared_ptr<TokenizerImpl> m_pimpl;
 };
 
+/**
+* @brief Returns an absolute path. The path is this library's directory
+ * concatenated with openvino_tokenizers OS specific
+ * * name (.so, .dll, .dylib, lib prefix). This is part of the interface
+ * because it's reused in Python bindings.
+ * tokenizers_relative_to_genai() and ScopedVar allow passing a path to
+ * openvino_tokenizers through env var removing one argument from
+ * Tokenizer's constructor.
+*/
+OPENVINO_GENAI_EXPORTS std::filesystem::path tokenizers_relative_to_genai();
+
+/**
+* @brief Sets ENVIRONMENT_VARIABLE_NAME to environment_variable_value
+ * and unsets in destructor. Does nothing if ENVIRONMENT_VARIABLE_NAME
+ * was already defined.
+*/
+class OPENVINO_GENAI_EXPORTS ScopedVar {
```

Comment:
should not be a part of public API.
You can put it into `src` and include `src` from python API

Reply:
but this header (inside `src`) will not be given to users, so they will not be able to use such API (at least easily).

Reply:
Why would we hide it?